MiniF/profiles/models.py

```diff
@@ -1,3 +1,43 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+class StartupProfile(models.Model):
+    """
+    Startup profile model.
+
+    Notes:
+    - Set the website field as URLField.
+    """
+    user_id = models.ForeignKey('users.UserProfile', on_delete=models.CASCADE)
+    company_name = models.CharField(max_length=255, unique=True)
+    description = models.TextField()
+    website = models.URLField(blank=True, null=True)
+    industry_id =models.IntegerField()
+    LOCATION_CHOICES = [
+        ("Cherkasy", "Cherkasy"),
+        ("Chernihiv", "Chernihiv"),
+        ("Chernivtsi", "Chernivtsi"),
+        ("Dnipro", "Dnipro"),
+        ("Donetsk", "Donetsk"),
+        ("Ivano-Frankivsk", "Ivano-Frankivsk"),
+        ("Kharkiv", "Kharkiv"),
+        ("Kherson", "Kherson"),
+        ("Khmelnytskyi", "Khmelnytskyi"),
+        ("Kirovohrad", "Kirovohrad"),
+        ("Kyiv", "Kyiv"),
+        ("Luhansk", "Luhansk"),
+        ("Lviv", "Lviv"),
+        ("Mykolaiv", "Mykolaiv"),
+        ("Odesa", "Odesa"),
+        ("Poltava", "Poltava"),
+        ("Rivne", "Rivne"),
+        ("Sumy", "Sumy"),
+        ("Ternopil", "Ternopil"),
+        ("Vinnytsia", "Vinnytsia"),
+        ("Volyn", "Volyn"),
+        ("Zakarpattia", "Zakarpattia"),
+        ("Zaporizhzhia", "Zaporizhzhia"),
+        ("Zhytomyr", "Zhytomyr"),]
```

Comment:
I suggest bringing the number of regions in line with Wikipedia

        https://uk.wikipedia.org/wiki/%D0%9E%D0%B1%D0%BB%D0%B0%D1%81%D1%82%D1%96_%D0%A3%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D0%B8